app/routes.js

```diff
@@ -1,7 +1,14 @@
 
 var Route = App.router;
+var Command = App.command;
 
 Route.get('/', function(req, res){
+    Command.run('inspire', {
```

Comment:
Please don't add this example of running commands from http routes inside '/' route.